src/Object.cpp

```diff
@@ -43,7 +41,7 @@ Object::~Object() {
 //}
 
 
-void Object::displayName(float font_size){
+void Object::displayName(const float &font_size, const osg::Vec3 &pos, const osg::Vec4 &color){
 
     if (!texttransform.valid()){
```

Comment:
`this->addChild(texttransform);` is executed on every invocation. Even if the child has been added before. What happens when the same child is added multiple times? Is it a problem?

Reply:
No, only if it was not created before, .valid() checks if the osg::ref_ptr is already set or not, only if it doesn't exist, it is added